README.md

```diff
@@ -447,6 +447,58 @@ If you would like to run a different linter, or even a custom version of eslint
 pass the `-n` or `--linter` argument with the path to the lint runner.  For example,
 if you plan to use jslint, you can install `lab-jslint` then pass `--linter node_modules/lab-jslint`.
 
+## Integration with an assertion library
+
+Using the `--assert` argument allows you to integrate Lab with your favourite assertion library. It works by
```

Comment:
Hey, I am an Aussie living in the UK so, sorry about that. You want me to change it?